public/typescript/__test__/gui.test.ts

```diff
@@ -21,3 +22,86 @@ it('Should show retry button when active', async () => {
   // Should clean up and hide the button again.
   expect(button.classList.contains('hidden')).toBeTruthy();
 });
+
+it('getErrorInformation', () => {
```

Comment:
We usually write our test in English, but keeping the test expectations in Dutch is fine with me.